plugins/base/frontend/src/main/components/search/dokkaFuzzyFilter.tsx

```diff
@@ -38,6 +39,11 @@ const highlightMatchedPhrases = (records: OptionWithSearchResult[]): OptionWithH
     })
 }
 
+const hasApproperateLenght = (searchResult: OptionWithSearchResult, length: number): boolean => {
```

Comment:
"Approperate" is not a word. I think you meant "appropriate".

Reply:
Fair point, as i read this PR again i think this name was flawed from the beginning. Changed to something i found more descriptive: `hasAnyMatchedPhraseLongerThan`